tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -83,7 +99,7 @@ def example_plot(ax, fontsize=12, nodec=False):
     example_plot(ax)
 
 ###############################################################################
-# Specifying `constrained_layout=True` in the call to `plt.subplots`
+# Specifying ``constrained_layout=True`` in the call to ``plt.subplots``
```

Comment:
Using `plt.subplots(..., constrained_layout=True)` causes ...

Reply:
The sentence wasn't written by me, but I actually like it as it is. Otherwise it's too much doubled

> Using `__code__` cause *x* to happen: `__code__`

Reply:
Ok, then leave it. The sentence feels quite verbose to me, but I won't start a discussion here.